README.md

```diff
@@ -158,6 +158,8 @@ Output:
 sim.stop()
 ```
 
+> Note that if you're not using `ipython`, the `if __name__ == "__main__":` guard is necessary, as mentioned in the [Python docs](https://docs.python.org/3/library/multiprocessing.html#:~:text=For%20an%20explanation%20of%20why%20the%20if%20__name__%20%3D%3D%20%27__main__%27%20part%20is%20necessary%2C%20see%20Programming%20guidelines.), when writing your own scripts.
```

Comment:
I would recommend we add the `__name__` guard to the code block and put the opposite note in (i.e. "Note that if you're using ipython, you can skip the `__name__` guard"). People tend to copy/paste code blocks and execute them.